pkgs/applications/version-management/gita/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+    version = "0.7.2";
+    pname = "gita";
+
+    src = python3Packages.fetchPypi {
+        inherit pname version;
+        sha256 = "05sv16m73slldsy73x9s4l3q0md5zpxzg9x9hm2gg6h1dfi6q1r0";
+    };
+
+    pythonPath = with python3Packages; [
+        pyyaml
+    ];
+
+    meta = {
```

Comment:
Usually we have `meta = with lib; {`, that way we don't need to write `lib.` in all options within meta that use lib.